app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -39,26 +47,25 @@
     3 ----> socialSitChanged;
     4 ----> isPrivateChanged;
     5 ----> imageChanged;
-    6 ----> geoLocationChanged;
+    6 ----> latitudeChanged;
+    7 ----> longitudeChanged;
     */
 
-    private ArrayList<Boolean> stateVector;
+    private static boolean[] stateVector;
 
 
     public Mood(){
```

Comment:
Is this constructor call really necessary? A Mood event shouldn't exist without a user